docs/usage/customization-guide.md

```diff
@@ -619,7 +619,8 @@ The following features are available for matching:
 | | |          **`nx_gzip`**                 | bool       | Nest Accelerator GZIP support is enabled
 | **`kernel.config`** | attribute |          |            | Kernel configuration options
 |                  |              | **`<config-flag>`** | string | Value of the kconfig option
-| **`kernel.loadedmodule`** | flag |         |            | Loaded kernel modules
+| **`kernel.loadedmodule`** | flag |         |            | Loaded dynamic kernel modules
+| **`kernel.enabledmodule`** | flag |        |            | All enabled kernel modules, i.e. union of loaded dynamic modules and modules built into the kernel
```

Comment:
```suggestion
| **`kernel.loadedmodule`** | flag |         |            | Kernel modules loaded on the node as reported by `/proc/modules`
| **`kernel.enabledmodule`** | flag |        |            | Kernel modules loaded on the node and available as built-ins as reported by `modules.builtin`. Does not include any out-of-tree modules that are not loaded.
```

Reply:
Out-of-tree modules are no special in this regard. Also, we report **loaded** and **enabled-by-builtin** modules so it should be kinda evident that we don't list any **disabled** modules so I'd suggest just to drop this _Does not include any out-of-tree modules that are not loaded._ Or then reword it to _Does not include any modules that are not loaded or built into the kernel._ which sounds a bit dumb after the first sentence

Reply:
> I'd suggest just to drop this

works for me

Reply:
K, so @mythi's suggestion would become

```suggestion
| **`kernel.loadedmodule`** | flag |         |            | Kernel modules loaded on the node as reported by `/proc/modules`
| **`kernel.enabledmodule`** | flag |        |            | Kernel modules loaded on the node and available as built-ins as reported by `modules.builtin`
```

I think this is good.

@AhmedGrati could I ask for this one more change? 😇

Reply:
Sure, will do it 👍